### ISSUE TITLE
3.9.0 not on PyPi

### ISSUE THREAD

**Issue by mojicae** [pycontribs/jira#1922]
Any idea when 3.9.0 will land on PyPi? I am currently only seeing 3.8.0

https://pypi.org/project/jira/

**Comment by mojicae** [comment]
Fix is here: https://github.com/pycontribs/jira/pull/1923

Also saw someone opened a PR for this last week, so if that gets merged, mine can be closed as well as this issue

**Comment by wulmer** [comment]
The problem is that the [release workflow](https://github.com/pycontribs/jira/actions/workflows/release.yml) is not working correctly due to missing 2FA setup: 

> 400 User 'pycontribs' does not have two-factor authentication            enabled. Please enable two-factor authentication before attempting to           upload to PyPI. See                                                             https://test.pypi.org/help/#two-factor-authentication for more                  information.

(from: https://github.com/pycontribs/jira/actions/runs/12073287832/job/33669085296)

**Comment by studioj** [comment]
@ssbarnea ?

**Comment by AH-Hammi** [comment]
@studioj yeah, I forgot to post my temporary workaround. Thank you for presenting here again :)
The thing is, that this option is not version stable. Therefore, this shouldn't be used in any kind of deployment. And I want to build a program with some new functionality.

**Comment by studioj** [comment]
> @studioj yeah, I forgot to post my temporary workaround. Thank you for presenting here again :)
> The thing is, that this option is not version stable. Therefore, this shouldn't be used in any kind of deployment. And I want to build a program with some new functionality.

We need the owner @ssbarnea to unblock this I think... I'll have a look at other pycontribs repos to see if I could get this unblocked.

You could pip install git+{url of tag} to lock it to the specific version

**Comment by gpflaum** [comment]
Paging @ssbarnea: It looks like you are the only one who can fix the release workflow.

**Comment by wblondel** [comment]
Any news?

**Comment by studioj** [comment]
@ssbarnea tried to get CI up and running again, but it seems we're not there yet

**Comment by jimcarreer** [comment]
> [@ssbarnea](https://github.com/ssbarnea) tried to get CI up and running again, but it seems we're not there yet

I can maybe try and help this weekend?

**Comment by studioj** [comment]
temporary fix available like so => also shown in the readme

`pip install https://github.com/pycontribs/jira/releases/download/3.10.1/jira-3.10.1-py3-none-any.whl`

**Comment by studioj** [comment]
https://pypi.org/project/jira/3.10.5
